detection-rules/attachment_html_smuggling_settimeout.yml

```diff
@@ -0,0 +1,21 @@
+name: "Attachment: HTML smuggling with setTimeout"
+description: |
+  Recursively scans files and archives to detect HTML smuggling techniques.
+references:
+  - "https://www.microsoft.com/security/blog/2021/11/11/html-smuggling-surges-highly-evasive-loader-technique-increasingly-used-in-banking-malware-targeted-attacks/"
+type: "rule"
+source: |
+  type.inbound
+  and any(attachments, .size <= 400 and
+      (
+          .file_extension in~ ("html", "htm", "7z","bz2","gz","rar","tar","zip","zipx","iso","img") or
+          .file_type == "html"
+      )
+      and any(beta.binexplode(.),
+          "setTimeout" in .scan.javascript.identifiers
+          and any(.scan.strings.strings, ilike(., "*location.href*"))
```

Comment:
it looks like this doesn't detect some examples in the linked reference. I also don't see any mention of setTimeout on the page. I guess that means the link is loosely related?

![codes-c.png](https://user-images.githubusercontent.com/31489089/172995830-0b015f29-d173-4ed5-b923-25174594f9ed.png)

![Fig4-HTML-page.png](https://user-images.githubusercontent.com/31489089/172995896-e90f2099-c249-4731-992e-d7a08fd7076f.png)

Reply:
good catch. this link actually has nothing to do with the rule, and was just a copy/paste from other HTML smuggling rules. will remove. or we could keep it if it's helpful to have _something_ about HTML smuggling

Reply:
removed from this PR and #135

Reply:
ah yeah. I also like having a link to _something_  like you said. your call on which approach to take. could also add a yaml comment about why the link makes sense